t/e2e/services/generic_api_test.go

```diff
@@ -0,0 +1,86 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/status-im/status-go/geth/params"
+	"github.com/status-im/status-go/sign"
+	"github.com/status-im/status-go/t/e2e"
+
+	. "github.com/status-im/status-go/t/utils"
+)
+
+type GenericRPCApiSuite struct {
+	e2e.BackendTestSuite
+}
+
+func (s *GenericRPCApiSuite) testAPIExported(method string, expectExported bool) {
```

Comment:
It's a bit odd as it's named `testAPIExported` which suggest that it expects the API method to be available but it also takes `expectExported` param.

Maybe just create two methods: `AssertAPIMethodExported()` and `AssertAPIMethodUnexported()`?